content/basics/ably.textile

```diff
@@ -0,0 +1,89 @@
+
+Read more in our "Pub/Sub section":/channels/.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel). You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Read more in our "Presence and occupancy section":/presence-occupancy/.
+
+h3(#message-storage-and-history). Message storage and History
+
+Ably's Message storage and History allows messages to be accessed after initially being sent. You can persist the last message sent using the rewind channel option or enable persistence of all messages for 24 hours for the free tier or 72 hours for paid tiers. These messages can be retrieved using the History API. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations/.
+
+By default messages are only stored for two minutes on Ably’s servers, these messages can be retrieved in the event of a lost connection using the History API.
+
+Read more in our "Message storage and history section":/storage-history/.
+
+h3(#push-notifications). Push Notifications
+
+Ably's Push Notifications feature allows messages to be pushed to subscribers without needing a device to maintain a connection to Ably. Push Notifications are commonly used to display visual notifications to users or launch background processes  for apps in a battery-efficient manner.
+
+Read more in our "Push Notifications section":/push/.
+
+h2(#infrastructure). About the Infrastructure
+
+To ensure unrivaled realtime messaging and data streaming performance, availability, and reliability the Ably Global Network has 7 geographically-distributed core routing datacenters and 307 edge acceleration points-of-presence(PoPs).
+
+Building on the Global Network Ably focuses on a seamless experience, that's why Ably's infrastructure is built around the "Four Pillars of Dependability":https://ably.com/four-pillars-of-dependability.
+
+h3(#performance). Performance
+
+By providing consistently minimized latency and bandwidth requirements developers know what to expect and to design towards. If you know the boundaries Ably will always operate within, you can design, build, and scale features around them, confident they’ll perform as expected under various conditions.
```

Comment:
> By providing consistently minimized latency and bandwidth requirements developers know...

By providing consistently minimized latency and bandwidth requirements, developers know...